fda-model/ead/simulation.cs

```diff
@@ -308,11 +308,15 @@ internal double FindTopOfLevee(UncertainPairedData uncertainPairedData)
             IPairedData leveePairedData = uncertainPairedData.SamplePairedData(meanRandomProvider.NextRandom());
             for (int i=0; i<leveePairedData.Xvals.Length; i++)
             {
-                if (leveePairedData.Xvals[i] == 1)
+                if (leveePairedData.Yvals[i] == 1)
                 {
-                    stageList.Add(leveePairedData.Yvals[i]);
+                    stageList.Add(leveePairedData.Xvals[i]);
                 }
             }
+            if (stageList.Count == 0)
```

Comment:
you could find the highest failure probability

Reply:
the highest failure probability must be 1

Reply:
I think this is particularly important because we are not asking for the top of levee elevation. FDA 1 puts prob failure = 1 at the top of levee elevation if the user does not. Without that information, we'd be left to extrapolation which feels unappealing.

Reply:
i dont think we need to extrapolate, it is percentage failure.

Reply:
im also possibly missing something here.